data/landing.yaml

```diff
@@ -20,12 +20,12 @@ hero:
   subtitle: Our goal is to create a community of software architecture experts, collaborate to share best practices, tackle common challenges with innovative solutions, and stay at the forefront of the ever-evolving technology landscape.
   info: "**Open Source** MIT Licensed."
 
-#  image:
-#    path: "images" # path to image under configured assets directory. default 'images'
-#    filename: "logo_image.png" # filename of your hero image (including file extension)
-#    alt: "Logo" # Optional but recommended
-#    boxShadow: true # default 'false' (excludes .svg images)
-#    rounded: true # round the image corners? default 'false' (excludes .svg images)
+  image:
+    path: "images" # path to image under configured assets directory. default 'images'
+    filename: "abstract-image.png" # filename of your hero image (including file extension)
+    alt: "Logo" # Optional but recommended
+    boxShadow: false # default 'false' (excludes .svg images)
+    rounded: true # round the image corners? default 'false' (excludes .svg images)
```

Comment:
Just wondering, is there any reason not to keep the default value here?

Reply:
It adds image into a square box with shadows
![image](https://github.com/user-attachments/assets/c45bb9d7-6baf-46d7-b462-2e12f3002001)

Reply:
The shadow is added by `boxShadow: false`. The `rounded: true` or `rounded: false` doesn't change anything in visualization cause the corners are transparent. Tested it locally.